src/plugins/Emojis/styling/EmojiPicker.scss

```
@@ -1,11 +1,389 @@
+        background-color: var(--str-chat__emoji-picker-hover-background-color);
+      }
+
+      &:focus-visible {
+        @include utils.focusable;
+      }
+    }
+  }
+}
+
+// Forced light theme (`pickerProps.theme='light'`).
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)